appland-navie/webview.js

```diff
@@ -62,6 +62,7 @@ export function mountWebview() {
     });
 
     app.$on('open-install-instructions', () => vscode.postMessage({command: 'open-install-instructions'}))
+    app.$on('open-location', (path) => vscode.postMessage({command: 'open-location', path}))
```

Comment:
Note than in an update coming soon, the enclosing directory of the path will also be provided. So only that directory will need to be searched for the file. You could include the directory parameter in this PR and treat it as optional?

Reply:
Just be aware that you can’t simply join the directory to the path to identity the file. The file path location within the directory may not be known exactly (eg Jvm recordings).

Reply:
You can see the WIP change here - https://github.com/getappmap/appmap-js/pull/1752/files